docs/Training-ML-Agents.md

```diff
@@ -190,6 +190,7 @@ example environments are included in the provided config file.
 | init_entcoef         | How much the agent should explore in the beginning of training.                                                                                                                         | SAC                      |
 | lambd                | The regularization parameter.                                                                                                                                                           | PPO                      |
 | learning_rate        | The initial learning rate for gradient descent.                                                                                                                                         | PPO, SAC             |
+| learning_rate_schedule | Determines how learning rate changes over time. Options: linear (default), constant | PPO, SAC |
```

Comment:
The defaults are actually different for PPO and SAC - would just leave out the options as they're specified within the PPO/SAC docs

Reply:
should we provide a link to the PPO/SAC docs here?

Reply:
I think there's a link at the top of the table

Reply:
changes made. can you please approve?